### PR TITLE
[WIP] Align RHEL7 C2S to updated CIS v2.1.1

### PR DESCRIPTION
**Pull request by shawndwells**

### CODE REVIEW

**Comment by shawndwells** [comment]
Updated CIS @ https://benchmarks.cisecurity.org/tools2/linux/CIS_Red_Hat_Enterprise_Linux_7_Benchmark_v2.1.1.pdf

File: RHEL/7/input/profiles/C2S.xml [hunk 1]
```diff
@@ -4,643 +4,443 @@
...
 
-<!-- 1.1.1 Create Separate Partition for /tmp (Scored) -->
+<!--	1	Initial Setup -->
+<!--	1.1	Filesystem Configuration -->
+
+<!--	1.1.1.1	Ensure mounting of cramfs filesystems is disabled (Scored) -->
+<select idref="kernel_module_cramfs_disabled" selected="true" />
+
+<!--	1.1.1.2	Ensure mounting of freevxfs filesystems is disabled (Scored) -->
+<selectt idref="kernel_module_freevxfs_disabled" selected="true" />
```
Comment (mpreisler) [inline RHEL/7/input/profiles/C2S.xml]: selectt

**Comment by shawndwells** [comment]
On 5/8/17 12:29 PM, Martin Preisler wrote:
> +
> +<selectt idref="kernel_module_freevxfs_disabled" selected="true" />
>
> selectt
>

thanks. fixed in 3117c6d

**Comment by redhatrises** [comment]
Ack. Fixed conflicts manually and merged.

**Pull request closed without merging**